src/Altinn-application-owner-system/Functions/Models/CloudEvent.cs

```diff
@@ -0,0 +1,94 @@
+using System;
+using System.Net.Mime;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace AltinnApplicationOwnerSystem.Functions.Models
+{
+    /// <summary>
+    /// Represents a cloud event. Based on CloudEvent: https://github.com/cloudevents/spec/blob/v1.0/spec.md.
+    /// </summary>
+    public class CloudEvent
+    {
+        /// <summary>
+        /// Gets or sets the id of the event.
+        /// </summary>
+        [JsonPropertyName("id")]
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the source of the event.
+        /// </summary>
+        [JsonPropertyName("source")]
+        public Uri Source { get; set; }
+
+        /// <summary>
+        /// Gets or sets the specification version of the event.
+        /// </summary>
+        [JsonPropertyName("specversion")]
+        public string SpecVersion { get; set; }
+
+        /// <summary>
+        /// Gets or sets the type of the event.
+        /// </summary>
+        [JsonPropertyName("type")]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Gets or sets the subject of the event.
+        /// </summary>
+        [JsonPropertyName("subject")]
+        public string Subject { get; set; }
+
+        /// <summary>
+        /// Gets or sets the time of the event.
+        /// </summary>
+        [JsonPropertyName("time")]
+        public DateTime? Time { get; set; }
+
+        /// <summary>
+        /// Gets or sets the alternative subject of the event.
+        /// </summary>
+        [JsonPropertyName("alternativesubject")]
```

Comment:
alernativeSubject

I see you are consistent with lower case, have we standardized on that or are we on camelCase?

Reply:
The standard has standardized on lowercase :)

> CloudEvents attribute names MUST consist of lower-case letters ('a' to 'z') or digits ('0' to '9') from the ASCII character set.

https://github.com/cloudevents/spec/blob/v1.0.1/spec.md#attribute-naming-convention